internal/controller/operatorconfigmap_controller.go

```diff
@@ -694,6 +720,20 @@ func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscription() erro
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) reconcileNoobaaOperatorSubscription() error {
+	noobaaSubscription, err := c.getSubscriptionByPackageName("noobaa-operator")
+	if err != nil {
+		return err
+	}
+	if c.subscriptionChannel != "" && c.subscriptionChannel != noobaaSubscription.Spec.Channel {
```

Comment:
This code is still pointing to the wrong subscription channel. The noobaa subscription channel is NOT the client subscription channel.

Reply:
@leelavg has assured me in regards of downstream the channel should be the same as client-operator and in regards to upstream we should open a tech-debt issue.

Reply:
Any Engineer (even a maintainer) assurance is not enough. We should code defensibly. 
The point is that these 2 channels are conceptually different things and the code should reflect that. 

As I already mentioned offline, the solution is simple. 
An env var to the operator that specifies the desired channel with a panic in case it is not set.  
Then we can set this by default on the operator deployment (as a default value) to have this injected via the downstream build process.

Reply:
@nb-ohad we agreed on raising an issue and raise a follow-up PR for this, isn't it?